contents/docs/advanced/proxy/cloudfront.mdx

```diff
@@ -37,11 +37,17 @@ export const diagram = "https://res.cloudinary.com/dmukukwp6/image/upload/postho
 9. Do not enable AWS Web Application Firewall (WAF).
 10. Click “Create distribution” at bottom of the page.
 11. Edit the Cloudfront Distribution and add the assets origin `us-assets.i.posthog.com` or `eu-assets.i.posthog.com`.
-14. Add another behavior the path will be `/static/*`.
-15. Under "Origin,", choose the one created above.
-17. Under “cache policy,” choose “origin-cors”
-18. Under “Origin request policy,” choose “CORS-CustomOrigin.”
-19. Under “Response headers policy,” choose “CORS-with-preflight-and-SecurityHeadersPolicy.”
+12. Add another behavior the path will be `/static/*`.
+13. Under "Origin,", choose the one created above.
+14. Under “cache policy,” choose “origin-cors”
+15. Under “Origin request policy,” click “Create origin request policy.” On the “Create origin request policy” page:
+  1. Name your policy "origin-request-policy"
+  2. Under "Headers," select "Include the following headers."
+  3. Under "Add header", select "Origin."
+  4. Under "Query strings", select "All."
+  5. Click “Create” at bottom of the page.
+16. Under Origin request policy,” choose "origin-request-policy"
```

Comment:
The most minor of nits, but I noticed all the other bullet points end with punctuation, but a couple did not.

```suggestion
  1. Name your policy "origin-request-policy."
  2. Under "Headers," select "Include the following headers."
  3. Under "Add header", select "Origin."
  4. Under "Query strings", select "All."
  5. Click “Create” at bottom of the page.
16. Under Origin request policy,” choose "origin-request-policy."
```